BYOL 3.0 Docs update (#862)
128technology/docs: docs/intro_installation_byol_azure_conductor.md

| Avaiability Set Name    | Name of the existing availability set within the same resource group and region as the VNet selected above where the device will be deployed. |
| Instance size        | Select the size of the VM in the field Instance Size. |
| Instance Name           | Provide a name to the VM for the Session Smart Conductor. |
| SSR Version                 | SSR software version installed on the instance. |
| Artifactory Username | User portion of the artifactory credentials used to install the SSR software. |
| Artifactory Token | Token for the artifactory credentials used to install the SSR software. |
| Managed Identity | The Azure Managed identity used to manage permissions for the router instance. |
| Virtual Network Name    | Name of the existing VNet where the Session Smart Router is going to be deployed. |
| Control Subnet Name  | The name of the management subnet within the VNet. |
| Control Allowed CIDR      | The IP CIDR range of the endpoints allowed to originate traffic to the Conductor's management interface in the management subnet.  |

> | Admin Allowed CIDR      | It allows for restricting reachability to the management interface of the router to a well known source IP address CIDR range. By default is set to 0.0.0.0/0 allowing every IP address to reach the management interface. Once the deployment completes, it is highly recommended to update the configuration of the network security group to allow only access from the source IP address/es where the Session Smart Router will be administered.  |

Reviewer: ```suggestion
| Admin Allowed CIDR      | Allows for restricting reachability to the management interface of the router to a well known source IP address CIDR range. Default is set to 0.0.0.0/0 allowing every IP address to reach the management interface. Once the deployment completes, it is highly recommended to update the configuration of the network security group to allow only access from the source IP address/es where the Session Smart Router will be administered.  |
```

When deploying the Session Smart Router using the templates referenced in this section, the following infrastructure elements are created automatically to assist with the deployment process:
* Virtual machine with the Session Smart image specified in the template.

> * The router is deployed with three network interfaces: public, private and optional management interfaces.

Reviewer: ```suggestion
* The router is deployed with three network interfaces: public, private, and optional management interfaces.
```

6. Answer the following questions to launch the deployment of a Conductor-managed Router. For a description of the parameters of the template, please refer to [Launch the Conductor-managed Template](#launch-the-conductor-managed-template).

- Where do you want to deploy it?

>   - Provide the location where the VNet exists in the **Location** field (for example: eastus). All available locations [here](https://azure.microsoft.com/en-us/global-infrastructure/locations). Note the name of the Location field is one word and all lowercase like eastus, westus, westeurope, eastasia, etc.

Reviewer: ```suggestion
  - Provide the location where the VNet exists in the **Location** field (for example: eastus). See [Locations](https://azure.microsoft.com/en-us/global-infrastructure/locations) for a full list of locations. Note the name of the Location field is one word and all lowercase like eastus, westus, westeurope, eastasia, etc.
```

| Primary Control IP | The primary IP address of the Conductor |
| Secondary Control IP | The secondary IP address of the Conductor |
| Artifactory Username | User portion of the artifactory credentials used to install the SSR software. |
| Artifactory Token | Token for the artifactory credentials used to install the SSR software. |
| Virtual Network Name    | Name of the existing VNet where the Session Smart Router is going to be deployed. |
| Public Subnet Name      | The name of the public subnet within the VNet. |
| Public Subnet Allowed CIDR     | Corresponds to the source IP CIDR range of the SSR/s at the data center/branch (outside the cloud) allowed to originate traffic to the public interface of the router. This field allows for defining a well defined and trusted IP address range. It is common to set this field to 0.0.0.0/0 for now, as the source IP addresses of the routers at the data center or branch (outside the cloud) are not known at this time. However, after the deployment and once these external IP addresses are known it is recommended to provision them in the corresponding security groups to increase the degree of security. |

> | Admin Allowed CIDR      | It allows for restricting reachability to the management interface of the router to a well known source IP address CIDR range. By default is set to 0.0.0.0/0 allowing every IP address to reach the interface. Once the deployment completes, it is highly recommended to update the configuration of the network security group to allow only access from the source IP address/es where the Session Smart Router will be administered. |

Reviewer: ```suggestion
| Admin Allowed CIDR      | Allows for restricting reachability to the management interface of the router to a well known source IP address CIDR range. Default is set to 0.0.0.0/0 allowing every IP address to reach the interface. Once the deployment completes, it is highly recommended to update the configuration of the network security group to allow only access from the source IP address/es where the Session Smart Router will be administered. |
```

| Private Subnet Name     | The name of the private subnet within the VNet. |

> | Private Subnet Allowed CIDR    | Corresponds to the source IP CIDR range of the internal workloads/endpoints allowed to originate traffic to the private interface of the router. This field allows for defining a well defined and trusted IP address range. By default is set to 0.0.0.0/0 to allow every workload/endpoint to communicate with the router.  |

Reviewer: ```suggestion
| Private Subnet Allowed CIDR    | Corresponds to the source IP CIDR range of the internal workloads/endpoints allowed to originate traffic to the private interface of the router. This field allows for defining a well defined and trusted IP address range. Default is set to 0.0.0.0/0 to allow every workload/endpoint to communicate with the router.  |
```
